### ISSUE TITLE
check internet connection pragmatically or test resource availability in air-gaped mode

### ISSUE THREAD

**Issue by verhulstm** [terminal-labs/rambo#161]

**Comment by nixjdm** [comment]
What do you have in mind that we do with this information once we perform the test?

**Comment by nixjdm** [comment]
Post discussion: We want a simple test for the presence of an internet connection, and error out gracefully with a warning message. Various things may break in the process of `rambo up` when an internet connection isn't present, depending on what is cached locally, so we should check once at the beginning of interaction.

**Comment by alan-cugler** [comment]
Is there a separate ticket for graceful exit regardless of errors? It's
been a problem in probably a dozen situations since I started working with
rambo.

The only thing in common with the errors is that they cause rambo to sit in
idle until ctrl+C is used.

**Comment by nixjdm** [comment]
@alan-cugler Yes: #150. One of the few tickets currently marked as [high priority](https://github.com/terminal-labs/rambo/labels/high%20priority).
